src/Card.js

```diff
@@ -2,19 +2,14 @@ import React from 'react'
 import PropTypes from 'prop-types'
```

Comment:
Any new card or card container should be part of the homepage cards repository, instead of being created here. Otherwise we run the risk of replicating work.